### ISSUE TITLE
Switching output for an app in the mixer seems to be inaccessible to keyboard-only or screen reader users

### ISSUE THREAD

**Issue by zersiax** [File-New-Project/EarTrumpet#413]
I am a blind developer using NVDA ( open source screen reader) to use the app and I can get most stuff done rather easily, but this is one thing I can't seem to be able to do.
Currently, I can , via the keyboard, set a hotkey to bring up the volume mixer. I can see what app is using what device, what volume it is set at, and I can change that volume using left and right arrow keys.
I can find no way to , using the keyboard, move an app from one device to another, though. I realize that might come across as a feature request, but for me this really is a bug that makes the program a whole lot less useful for my needs.
If I am simply missing how this could be done I'd of course love to hear it :-)

**Comment by riverar** [comment]
@zersiax Hi there, accessibility is important to us so any issues in that area would be considered a bug, not a feature request. Before we go any further, please check if you're on Windows 10 build 17134 or higher. (This is the version of Windows 10 that introduces the audio redirection APIs that we use.) You can upgrade via the upgrade assistant https://www.microsoft.com/en-us/software-download/windows10 if you're behind.

Thanks!

**Comment by zersiax** [comment]
Currently on Insider build 18970.005. 

Mainly, the issue currently is that I honestly wouldn't know how to redirect an app. In the settings experience that is native to Windows, each app has an associated dropdown to select its input and output device, but I can't seem to find the mechanism for redirecting in EarTrumpet. Are you supposed to drag and drop the apps? If so, we'd need to find a keyboard equivalent to that

**Comment by riverar** [comment]
@zersiax Ah. Right-click an app, left-click the arrows, then select a device.

Example:
![Audio redirection example GIF](https://github.com/File-New-Project/EarTrumpet/raw/master/Graphics/hero4.gif)

**Comment by davuxcom** [comment]
@zersiax Thanks for using EarTrumpet!  We very much would like to hear about your annoyances and blocking issues. Getting accessibility right can be difficult, we appreciate having a subject matter expert actually exercise the UI in the wild. Please to let us know if you have other blockers, it's supposed to work well enough with a screen reader.

EarTrumpet flyout is a list of devices, with a further list of apps on that device.  Each app can be interacted with by pressing <kbd>space</kbd> to open a popup UI which has the device switching options.

You should be able to use this pattern:
1. Type your "Open Flyout" hotkey and the flyout opens (focus goes to the device volume and you can press left/right to change the volume)
2. Type <kbd>tab</kbd> or <kbd>down arrow</kbd> to focus volume sliders for each app
3. Type <kbd>space</kbd> to open the popup for that app. This popup is modal, it has a grouping of volume sliders if the app presents more than one, and it has a toolbar with "Switch Devices" and "Close". 
4. Type tab to set focus to the Switch Devices button (it will jump between Close, Switch Devices, and the app list--again it's modal so you won't see your other apps once you enter a popup)
5. Type <kbd>enter</kbd> to activate the menu, and then use up/down to select a device.  Having to press <kbd>enter</kbd> instead of the universal <kbd>space</kbd> key is a bug that we'll get fixed up.

Once you activate a device on the menu you can press <kbd>Escape</kbd> once to exit the popup and once more to exit the flyout.

**Comment by riverar** [comment]
Hey @zersiax following up -- did this help?
